CONTRIBUTING.md

```diff
@@ -33,6 +33,59 @@ For example, see [mongo-go-driver tests configuration](https://github.com/Ferret
 `TEST` environment variable should have the value `mongo-go-driver`, or be empty.
 It defines what test configuration to run; empty value runs all configurations.
 
+### Running custom tests
+
+In order to test your application with FerretDB you must use the `command` runner and add your repository as a submodule.
+The `command` runner will invoke any command and CLI arguments.
+
+For example if you want to test your Node.js application with FerretDB, you would do the following:
+
+1. Add the submodule to dance `git submodule add https://github.com/my-app`
+2. Create a shell script in the `tests` directory called `my-app-runtime.sh` with the required logic needed for your test
+3. Create a YAML file called `my-app.yml` in the `tests` directory and provide the `args` field with the shell script so that the runner can invoke it
```

Comment:
```suggestion
3. Create a YAML file called `my-app.yml` in the `tests` directory and provide the `args` field with the shell script so that the runner can invoke it.
```